examples/repl.rs

```diff
@@ -390,6 +398,24 @@ mod example {
                 }
             }
 
+            Some("break") | Some("b") => {
+                context.load_debuginfo_if_necessary()?;
+                if let Some(input) = parts.next() {
+                    if let Ok(addr) = str::parse::<usize>(input) {
+                        context.mut_remote()?.set_breakpoint(addr)?;
+                    } else if let Some(symbol) = context.debuginfo().get_symbol_address(input) {
+                        context.mut_remote()?.set_breakpoint(symbol)?;
+                        println!("Set breakpoint at '{}' ({:#016x})", input, symbol);
+                    } else {
+                        Err(format!("Could not set breakpoint"))?
+                    }
+                } else {
+                    Err(format!(
+                        "Breakpoints need to be set on a symbol or a given address."
```

Comment:
nit: It might be even more helpful to give an example as how to set a break-point. What do you think ? Or we can add it to help and then point them to it here ?